run_inference.sh

```diff
@@ -1,17 +1,18 @@
 # rationale generation
-CUDA_VISIBLE_DEVICES=0,1 python main.py \
-    --model allenai/unifiedqa-t5-base \
-    --user_msg rationale --img_type detr \
-    --bs 8 --eval_bs 4 --eval_acc 10 --output_len 512 \
-    --final_eval --prompt_format QCM-LE \
-    --evaluate_dir models/rationale
+CUDA_VISIBLE_DEVICES=0,1 python src/main.py \
+  --model allenai/unifiedqa-t5-base \
+  --user_msg rationale \
+  --img_type detr \
+  --bs 8 \
+  --eval_bs 4 \
+  --eval_acc 10 \
+  --output_len 512 \
+  --final_eval \
+  --prompt_format QCM-LE \
+  --evaluate_dir models/rationale \
+  --data_root data/dataset \
+  --caption_file data/dataset/captions.json \
+  --evaluate_dir models/MM-CoT-UnifiedQA-base-Rationa
```

Comment:
This changes allow the script execution because the default structure was changed.
We can change the default value of the arguments as follow:
data_root: from 'data' to 'data/dataset'
caption_file: from 'data/captions.json' to 'data/dataset/captions.json'

The last one, the evaluate_dir is necessary but it should be fixed because the rigth args is
--evaluate_dir models/MM-CoT-UnifiedQA-base-Rationa**le**

Reply:
Do you think it is better to update the args default value or to leave this script parameters?
If we change args default value there is no need to modify the other sh scripts

Reply:
Ok, fix the last parameter and you can keep the others